pallets/committee/src/lib.rs

```diff
@@ -524,7 +533,7 @@ pub mod pallet {
 				Error::<T>::InvalidVotingPeriod
 			);
 
-			VotingPeriod::<T>::set(voting_period);
+			PendingVotingPeriod::<T>::set(Some(voting_period));
```

Comment:
we should add events for that as well, one for Pending with `voting_period` and when it takes effect, one for when it's set in the `upkeep` function